docs/development/workflow/get_devel_version.rst

```diff
@@ -222,30 +222,36 @@ of `Astropy`_, for now this straightforward way will work (if you want to jump
 ahead to the more sophisticated method look at :ref:`virtual_envs`).
 
 .. note::
-    There are a couple of circumstances in which this quick method of
-    activating your copy of `Astropy`_ will NOT work and you need to go
-    straight to using a virtual python environment:
-
-    + You use Python 3.
-    + You want to work on C or Cython code in `Astropy`_.
+    If you want to work on C or Cython code in `Astropy`_, this quick method
+    of activating your copy of `Astropy`_ will *not* work _ -- you need to go 
+    straight to using a virtual python environment.
+
+If you have decided to use the recommended "activation" method with
+``pip``, please note the following: Before trying to install, 
+check that you have the required dependencies: "cython" and "jinja2". 
+If not, install them with ``pip``. Please note that by ``pip`` 
```

Comment:
I think the ``Please note that...`` onwards can just be replaced by:

*Note that on some platforms, the pip command is ``pip3`` instead of ``pip``, so be sure to use this instead in the examples below if that is the case*

Reply:
Yup, changed...